templates/terraform/resource.erb

```diff
@@ -309,14 +358,38 @@ func resource<%= resource_name -%>Import(d *schema.ResourceData, meta interface{
 		return nil, fmt.Errorf("Error constructing id: %s", err)
 	}
 	d.SetId(id)
+  <%= lines(compile(object.custom_code.post_import)) if object.custom_code.post_import -%>
 
 	return []*schema.ResourceData{d}, nil
+<% end -%>
 }
 
 <% properties.each do |prop| -%>
+<% if object.custom_code.custom_flatten.key? prop.name -%>
+<%= lines(compile_template(object.custom_code.custom_flatten[prop.name],
+                           prefix: resource_name,
+                           prop: prop)) -%>
+<% else -%>
 <%= lines(build_flatten_method(resource_name, prop), 1) -%>
 <% end -%>
+<% end -%>
 
 <% settable_properties.each do |prop| -%>
+<% if object.custom_code.custom_expand.key? prop.name -%>
+  <%= lines(compile_template(object.custom_code.custom_expand[prop.name],
+                           prefix: resource_name,
```

Comment:
nit: I think this should be indented another couple of spaces

Reply:
:+1: